server/middleware/setUpStaticResources.ts

```diff
@@ -17,8 +17,8 @@ export default function setUpStaticResources(): Router {
     '/assets',
     '/assets/stylesheets',
     '/assets/js',
-    '/node_modules/govuk-frontend/govuk/assets',
-    '/node_modules/govuk-frontend',
+    '/node_modules/govuk-frontend/dist/govuk/assets',
```

Comment:
to match https://github.com/ministryofjustice/hmpps-template-typescript/blob/eb86ef136745d50101ef8f902f5c3476a5f4ceb8/server/middleware/setUpStaticResources.ts#L17